great_ape_safe/ape_api/ebtc.py

```diff
@@ -1287,14 +1472,14 @@ def close_cdp(self, cdp_id):
             cdp_id_coll,
             cdp_id_stake,
             cdp_id_liq_reward_shares,
-            _,
-            _,
+            cdp_id_status,
         ) = self.cdp_manager.Cdps(cdp_id)
         assert self.cdp_manager.cdpStEthFeePerUnitIndex(cdp_id) == 0
         assert cdp_id_debt == 0
         assert cdp_id_coll == 0
         assert cdp_id_stake == 0
         assert cdp_id_liq_reward_shares == 0
+        assert cdp_id_status == CdpStatus.CLOSED.value
```

Comment:
duplicate assert as in L1456, no?

btw, something seems off w/ the lint. it added some noise on the commit diff and seems not passing the ci lint check

Reply:
Good call, fixed. With regards to the lint issue, I am looking into it, lint check passes locally.